README.rst

```diff
@@ -444,6 +444,26 @@ Usage with Django REST Framework
 Make sure that ``djmoney`` is in the ``INSTALLED_APPS`` of your ``settings.py`` and MoneyFields to automatically
 work with Django REST Framework.
 
+You can add a serializable field the following way:
+
+.. code:: python
+
+    from djmoney.contrib.django_rest_framework import MoneyField
+
+    class Serializers(serializers.Serializer):
+        my_computed_prop = MoneyField(max_digits=10, decimal_places=2)
```

Comment:
Adding this part makes sense I guess, to show how you can setup non-model serializers as well :+1: